test/parallel/test-stdout-close-catch.js

```diff
@@ -1,27 +1,28 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
-var child_process = require('child_process');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const child_process = require('child_process');
 
-var testScript = path.join(common.fixturesDir, 'catch-stdout-error.js');
+const testScript = path.join(common.fixturesDir, 'catch-stdout-error.js');
 
-var cmd = JSON.stringify(process.execPath) + ' ' +
-          JSON.stringify(testScript) + ' | ' +
-          JSON.stringify(process.execPath) + ' ' +
-          '-pe "process.stdin.on(\'data\' , () => process.exit(1))"';
+const cmd = JSON.stringify(process.execPath) + ' ' +
+            JSON.stringify(testScript) + ' | ' +
+            JSON.stringify(process.execPath) + ' ' +
+            '-pe "process.stdin.on(\'data\' , () => process.exit(1))"';
 
-var child = child_process.exec(cmd);
-var output = '';
-var outputExpect = { 'code': 'EPIPE',
+const child = child_process.exec(cmd);
+let output = '';
+const outputExpect = { 'code': 'EPIPE',
                      'errno': 'EPIPE',
```

Comment:
@cjihrig - thanks for looking at this.
I'm perfectly happy to align to any coding style, however, what is the style to be? make lint didn't make a peep over the alignment of these lines. My own style would be to indent these lines like so:
```
const outputExpect = {
  'code': 'EPIPE',
  'errno': 'EPIPE',
  'syscall': 'write'
};
```
but I don't know if that's the indentation you'd like to see.